src/olympia/devhub/forms.py

```diff
@@ -96,6 +96,18 @@ def clean_slug(self):
 
 class LicenseRadioSelect(forms.RadioSelect):
 
+    def get_context(self, name, value, attrs):
+        context = super(LicenseRadioSelect, self).get_context(
+            name, value, attrs)
+
+        # Make sure the `class` is only set on the radio fields and
+        # not on the `ul`. This avoids style issues among other things.
+        # See https://github.com/mozilla/addons-server/issues/8902
+        # and https://github.com/mozilla/addons-server/issues/8920
+        del context['widget']['attrs']['class']
+
+        return context
```

Comment:
This is fine as a stopgap fix but IMHO our styles should cope with the way django sets up the class instead of having to randomly delete an attribute on the widget inside `get_context()` ...

Reply:
I totally agree, this is only a workaround that can be cherry-picked. It would be a CSS and JavaScript fix and especially with the JS I'm always scared on AMO with cherry-picking things and not knowing what's going on.

I'll file a follow-up issue.